docs/getting-started.md

```diff
@@ -6,24 +6,30 @@ prerequisites:
 
 1. NodeJS (version 14 or higher) and `npm`.
 2. `git`.
-3. Optional: A working [Docker][1] installation. The server can also be run
-   straight from the cli.
+3. MySQL (For Mac: install via ).
+    - For Mac: easy install Homebrew [MySQL package](https://formulae.brew.sh/formula/mysql) via [Homebrew](https://brew.sh/).
+    - For Linux: install [MySQL Community Version](https://www.mysql.com/products/community/).
```

Comment:
Would suggest something like:

```suggestion
    - For Linux: `apt install mysql-server`
```

Everyone on linux will likely want to do this via a package manager.